mlflow-tests/src/mlflow_tests/managers/k8s/service_account.py

```
@@ -0,0 +1,392 @@
+        try:
+            logger.info(f"Creating service account '{name}' in namespace '{namespace}'")
+            self.core_v1_api.create_namespaced_service_account(
+                namespace=namespace, body=service_account
+            )
+            # Wait for service account to be ready before proceeding
+            time.sleep(1)
+
+            # Verify the service account exists and is ready
+            self.core_v1_api.read_namespaced_service_account(name=name, namespace=namespace)
+            logger.info(f"Service account '{name}' created successfully")
+            return None
```

Ruff (0.14.14)

Consider moving this statement to an `else` block

(TRY300)